staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2341,6 +2341,9 @@ const (
 	TerminationMessagePathDefault string = "/dev/termination-log"
 )
 
+// A single application container that you want to run within a pod.
```

Comment:
This PR *depends* on this comment being EXACTLY the same as `Container` because our tooling mis-processes what you have done here, and takes the `InitContainer` comment for the `Container` docs.

https://github.com/kubernetes/kubernetes/pull/115364#issuecomment-1421689663

I don't think we should do this sort of trick unless we fix the tools to generate proper docs - it's too fragile.

Reply:
To be clear - I bet that it would not take long (O(hours)?) to find the place in the gengo stack which is mis-processing this.  I don't know if fixing it would be easy or not.